invenio_communities/communities/records/systemfields/parent_community.py

```diff
@@ -51,6 +51,7 @@ def set_obj(self, record, obj):
         # Check if obj is None and remove 'parent' key from record
         if obj is None:
             record.pop("parent", None)
+            self._set_cache(record, None)
```

Comment:
Nice! I suspect that might be the case in other system fields that don't clear their cache... Can you grep-around maybe to see if it's true?